app/models/concerns/offer_filtering.rb

```diff
@@ -47,5 +49,11 @@ def self.last_6pm
       t -= 24.hours if now < t
       t
     end
+
+    def self.assoicate_delivery_and_schedule
+      res = joins("LEFT OUTER JOIN deliveries ON offers.id = deliveries.offer_id")
+      res = res.joins("LEFT OUTER JOIN schedules ON deliveries.schedule_id = schedules.id")
+      res
```

Comment:
```ruby
def self.assoicate_delivery_and_schedule
   joins <<-SQL
      LEFT OUTER JOIN deliveries ON offers.id = deliveries.offer_id
      LEFT OUTER JOIN schedules ON deliveries.schedule_id = schedules.id
   SQL
end
```
absolutely nothing wrong with this function, just thought I'd share my "esthetic" preferences : )

Reply:
(don't change anything, just for the sake of sharing)